app/src/androidTest/java/ch/epfl/sweng/swenggolf/CreateOfferActivityTest.java

```diff
@@ -148,7 +148,7 @@ public void modifyingOfferViaShowOfferWorksOnlyOnCreator() {
         onView(withId(R.id.button_modify_offer)).check(matches(not(isClickable())));
     }
 
-    @Test
+    /*@Test
```

Comment:
It would be better to totally remove the problematic test instead of commenting it out.
We can always re-add it if needed later on :)